diff -rc cache/factions/drakes.cfg editable/factions/drakes.cfg
*** cache/factions/drakes.cfg	2012-01-31 11:41:12.455939865 +0100
--- editable/factions/drakes.cfg	2012-01-31 11:41:13.237929082 +0100
***************
*** 5,12 ****
      image="units/drakes/flare.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      type=Drake Flare
!     leader=Drake Flare,Fire Drake,Drake Arbiter,Drake Thrasher,Drake Warrior,Saurian Oracle,Saurian Soothsayer
!     random_leader=Drake Flare,Fire Drake,Drake Arbiter,Drake Thrasher,Drake Warrior
      recruit=Drake Burner,Drake Clasher,Drake Glider,Drake Fighter,Saurian Skirmisher,Saurian Augur
      terrain_liked=Hh, Ha, Ds, Ss
  [/multiplayer_side]
--- 5,11 ----
      image="units/drakes/flare.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      type=Drake Flare
!     leader=Drake Flare,Drake Warrior,Fire Drake
      recruit=Drake Burner,Drake Clasher,Drake Glider,Drake Fighter,Saurian Skirmisher,Saurian Augur
      terrain_liked=Hh, Ha, Ds, Ss
  [/multiplayer_side]
diff -rc cache/factions/knalgans.cfg editable/factions/knalgans.cfg
*** cache/factions/knalgans.cfg	2012-01-31 11:41:12.455939865 +0100
--- editable/factions/knalgans.cfg	2012-01-31 11:41:13.238929068 +0100
***************
*** 2,19 ****
  [multiplayer_side]
      id=Knalgan Alliance
      name=_"Knalgan Alliance"
!     image="units/dwarves/steelclad.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     type=Dwarvish Steelclad
!     leader=Dwarvish Steelclad,Dwarvish Thunderguard,Dwarvish Stalwart,Rogue,Trapper
      recruit=Dwarvish Guardsman,Dwarvish Fighter,Dwarvish Ulfserker,Dwarvish Thunderer,Thief,Poacher,Footpad,Gryphon Rider
      terrain_liked=Hh, Ha, Mm
-     [ai]
-         [aspect]
-             id=recruitment_pattern
-             [facet]
-                 value=fighter,fighter,fighter,mixed fighter,archer,scout
-             [/facet]
-         [/aspect]
-     [/ai]
  [/multiplayer_side]
--- 2,11 ----
  [multiplayer_side]
      id=Knalgan Alliance
      name=_"Knalgan Alliance"
!     image="units/dwarves/runesmith.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     type=Dwarvish Runesmith
!     leader=Dwarvish Runesmith,Outlaw,Rogue
      recruit=Dwarvish Guardsman,Dwarvish Fighter,Dwarvish Ulfserker,Dwarvish Thunderer,Thief,Poacher,Footpad,Gryphon Rider
      terrain_liked=Hh, Ha, Mm
  [/multiplayer_side]
diff -rc cache/factions/loyalists.cfg editable/factions/loyalists.cfg
*** cache/factions/loyalists.cfg	2012-01-31 11:41:12.455939865 +0100
--- editable/factions/loyalists.cfg	2012-01-31 11:41:13.287928394 +0100
***************
*** 2,15 ****
  [multiplayer_side]
      id=Loyalists
      name= _"Loyalists"
!     image="units/human-loyalists/lieutenant.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     type=Lieutenant
!     leader=Lieutenant,Swordsman,Pikeman,Javelineer,Shock Trooper,Longbowman,White Mage,Red Mage
!     random_leader=Lieutenant,Swordsman,Pikeman,Javelineer,Longbowman,White Mage,Red Mage
      recruit=Cavalryman,Horseman,Spearman,Fencer,Heavy Infantryman,Bowman,Mage,Merman Fighter
      terrain_liked=Gg, Ww, Wo
-     [ai]
-         recruitment_pattern=fighter,fighter,fighter,archer,mixed fighter,scout
-     [/ai]
  [/multiplayer_side]
--- 2,11 ----
  [multiplayer_side]
      id=Loyalists
      name= _"Loyalists"
!     image="units/human-loyalists/duelist.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     type=Duelist
!     leader=Duelist,Lieutenant,Red Mage
      recruit=Cavalryman,Horseman,Spearman,Fencer,Heavy Infantryman,Bowman,Mage,Merman Fighter
      terrain_liked=Gg, Ww, Wo
  [/multiplayer_side]
diff -rc cache/factions/northerners.cfg editable/factions/northerners.cfg
*** cache/factions/northerners.cfg	2012-01-31 11:41:12.456939851 +0100
--- editable/factions/northerners.cfg	2012-01-31 11:41:13.287928394 +0100
***************
*** 2,14 ****
  [multiplayer_side]
      id=Northerners
      name= _"Northerners"
!     image="units/orcs/warrior.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     type=Orcish Warrior
!     leader=Orcish Warrior,Troll,Troll Rocklobber,Orcish Crossbowman,Orcish Slayer
      recruit=Orcish Grunt,Troll Whelp,Wolf Rider,Orcish Archer,Orcish Assassin,Naga Fighter,Goblin Spearman
      terrain_liked=Hh, Ha, Mm, Ss
-     [ai]
-         recruitment_pattern=fighter,fighter,fighter,mixed fighter,archer,scout
-     [/ai]
  [/multiplayer_side]
--- 2,11 ----
  [multiplayer_side]
      id=Northerners
      name= _"Northerners"
!     image="units/orcs/ruler.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     type=Orcish Ruler
!     leader=Orcish Ruler,Troll Hero,Troll Shaman
      recruit=Orcish Grunt,Troll Whelp,Wolf Rider,Orcish Archer,Orcish Assassin,Naga Fighter,Goblin Spearman
      terrain_liked=Hh, Ha, Mm, Ss
  [/multiplayer_side]
diff -rc cache/factions/rebels.cfg editable/factions/rebels.cfg
*** cache/factions/rebels.cfg	2012-01-31 11:41:12.456939851 +0100
--- editable/factions/rebels.cfg	2012-01-31 11:41:13.288928380 +0100
***************
*** 5,15 ****
      image="units/elves-wood/captain.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      type=Elvish Captain
!     leader=Elvish Captain,Elvish Hero,Elvish Ranger,Elvish Marksman,Elvish Druid,Elvish Sorceress,White Mage,Red Mage,Elder Wose
!     random_leader=Elvish Captain,Elvish Hero,Elvish Ranger,Elvish Marksman,Elvish Druid,Elvish Sorceress,White Mage,Red Mage
      recruit=Elvish Fighter,Elvish Archer,Mage,Elvish Shaman,Elvish Scout,Wose,Merman Hunter
      terrain_liked=Ff, Fa, Ss, Ww
-     [ai]
-         recruitment_pattern=fighter,fighter,archer,archer,mixed fighter,healer,scout
-     [/ai]
  [/multiplayer_side]
--- 5,11 ----
      image="units/elves-wood/captain.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      type=Elvish Captain
!     leader=Elvish Captain,Elvish Lord,Elvish Ranger
      recruit=Elvish Fighter,Elvish Archer,Mage,Elvish Shaman,Elvish Scout,Wose,Merman Hunter
      terrain_liked=Ff, Fa, Ss, Ww
  [/multiplayer_side]
diff -rc cache/factions/undead.cfg editable/factions/undead.cfg
*** cache/factions/undead.cfg	2012-01-31 11:41:12.456939851 +0100
--- editable/factions/undead.cfg	2012-01-31 11:41:13.288928380 +0100
***************
*** 5,15 ****
      image="units/undead-necromancers/dark-sorcerer.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      type=Dark Sorcerer
!     leader=Dark Sorcerer,Revenant,Deathblade,Bone Shooter,Necrophage
!     random_leader=Dark Sorcerer,Revenant,Deathblade,Bone Shooter
      recruit=Skeleton,Skeleton Archer,Walking Corpse,Ghost,Vampire Bat,Dark Adept,Ghoul
      terrain_liked=Ss, Hh, Ha, Ww
-     [ai]
-         recruitment_pattern=fighter,fighter,archer,fighter,scout,archer
-     [/ai]
  [/multiplayer_side]
--- 5,11 ----
      image="units/undead-necromancers/dark-sorcerer.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      type=Dark Sorcerer
!     leader=Dark Sorcerer,Death Knight,Deathblade
      recruit=Skeleton,Skeleton Archer,Walking Corpse,Ghost,Vampire Bat,Dark Adept,Ghoul
      terrain_liked=Ss, Hh, Ha, Ww
  [/multiplayer_side]
diff -rc cache/units/drakes/Glider.cfg editable/units/drakes/Glider.cfg
*** cache/units/drakes/Glider.cfg	2012-01-31 11:41:12.473939617 +0100
--- editable/units/drakes/Glider.cfg	2012-01-31 11:41:13.289928366 +0100
***************
*** 7,13 ****
      {MAGENTA_IS_THE_TEAM_COLOR}
      ellipse="misc/ellipse"
      profile="portraits/drakes/glider.png"
!     hitpoints=32
      movement_type=drakeglide
      movement=8
      experience=35
--- 7,13 ----
      {MAGENTA_IS_THE_TEAM_COLOR}
      ellipse="misc/ellipse"
      profile="portraits/drakes/glider.png"
!     hitpoints=33
      movement_type=drakeglide
      movement=8
      experience=35
diff -rc cache/units/dwarves/Runesmith.cfg editable/units/dwarves/Runesmith.cfg
*** cache/units/dwarves/Runesmith.cfg	2012-01-31 11:41:12.466939711 +0100
--- editable/units/dwarves/Runesmith.cfg	2012-01-31 11:43:55.918695620 +0100
***************
*** 7,18 ****
      image=units/dwarves/runesmith.png
      profile="portraits/dwarves/runemaster.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=40
      movement_type=dwarvishfoot
      [resistance]
          blade=60
!         pierce=70
!         impact=70
      [/resistance]
      movement=4
      level=2
--- 7,21 ----
      image=units/dwarves/runesmith.png
      profile="portraits/dwarves/runemaster.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=42
      movement_type=dwarvishfoot
      [resistance]
+         arcane=80
          blade=60
!         cold=80
!         fire=80
!         pierce=60
!         impact=60
      [/resistance]
      movement=4
      level=2
***************
*** 45,53 ****
          name=hammer
          description= _"hammer"
          icon=attacks/hammer-dwarven-runic.png
          type=impact
          range=melee
!         damage=12
          number=2
          [specials]
              {WEAPON_SPECIAL_MAGICAL}
--- 48,68 ----
          name=hammer
          description= _"hammer"
          icon=attacks/hammer-dwarven-runic.png
+         type=cold
+         range=melee
+         damage=9
+         number=2
+         [specials]
+             {WEAPON_SPECIAL_MAGICAL}
+         [/specials]
+     [/attack]
+     [attack]
+         name=hammer
+         description= _"hammer"
+         icon=attacks/hammer-dwarven-runic.png
          type=impact
          range=melee
!         damage=9
          number=2
          [specials]
              {WEAPON_SPECIAL_MAGICAL}
diff -rc cache/units/elves/Druid.cfg editable/units/elves/Druid.cfg
*** cache/units/elves/Druid.cfg	2012-01-31 11:41:12.469939672 +0100
--- editable/units/elves/Druid.cfg	2012-01-31 11:41:13.290928352 +0100
***************
*** 18,24 ****
      usage=healer
      description= _ "The magic of the wood-elves is poorly suited for combat, but effective nonetheless. The forests in which they thrive can become quickened by a word of command, and will lash out at those who threaten their peace.
  
! The chief ability of Druids lies in healing, and it is for this skill that they are revered by their people."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_CURES}+{SPECIAL_NOTES_MAGICAL}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
--- 18,24 ----
      usage=healer
      description= _ "The magic of the wood-elves is poorly suited for combat, but effective nonetheless. The forests in which they thrive can become quickened by a word of command, and will lash out at those who threaten their peace.
  
! The chief ability of Druids lies in healing, and it is for this skill that they are revered by their people."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_CURES}+{SPECIAL_NOTES_MAGICAL}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
***************
*** 49,54 ****
--- 49,55 ----
          description=_"ensnare"
          type=impact
          [specials]
+             {WEAPON_SPECIAL_MARKSMAN}
              {WEAPON_SPECIAL_SLOW}
          [/specials]
          damage=6
diff -rc cache/units/elves/Enchantress.cfg editable/units/elves/Enchantress.cfg
*** cache/units/elves/Enchantress.cfg	2012-01-31 11:41:12.470939659 +0100
--- editable/units/elves/Enchantress.cfg	2012-01-31 11:41:13.291928338 +0100
***************
*** 16,22 ****
      advances_to=Elvish Sylph
      cost=55
      usage=mixed fighter
!     description= _ "The magic of the elves is wholly unlike that of humanity, so much so that humans are almost universally incapable of understanding it, let alone using it. Likewise, those who are masters of it are equally enigmatic; though it can be said that they are revered by their people, very little can be said about their actual function or purpose within that society."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_ARCANE}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
--- 16,22 ----
      advances_to=Elvish Sylph
      cost=55
      usage=mixed fighter
!     description= _ "The magic of the elves is wholly unlike that of humanity, so much so that humans are almost universally incapable of understanding it, let alone using it. Likewise, those who are masters of it are equally enigmatic; though it can be said that they are revered by their people, very little can be said about their actual function or purpose within that society."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_MARKSMAN}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_ARCANE}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
***************
*** 48,53 ****
--- 48,54 ----
          description=_"entangle"
          type=impact
          [specials]
+             {WEAPON_SPECIAL_MARKSMAN}
              {WEAPON_SPECIAL_SLOW}
          [/specials]
          damage=5
diff -rc cache/units/elves/Lord.cfg editable/units/elves/Lord.cfg
*** cache/units/elves/Lord.cfg	2012-01-31 11:41:12.468939685 +0100
--- editable/units/elves/Lord.cfg	2012-01-31 11:41:13.291928338 +0100
***************
*** 41,54 ****
          icon=attacks/sword-elven.png
          type=blade
          range=melee
!         damage=8
          number=4
      [/attack]
      [attack]
          name=faerie fire
          description= _"faerie fire"
          type=arcane
!         damage=7
          number=3
          range=ranged
          [specials]
--- 41,54 ----
          icon=attacks/sword-elven.png
          type=blade
          range=melee
!         damage=7
          number=4
      [/attack]
      [attack]
          name=faerie fire
          description= _"faerie fire"
          type=arcane
!         damage=6
          number=3
          range=ranged
          [specials]
diff -rc cache/units/elves/Shaman.cfg editable/units/elves/Shaman.cfg
*** cache/units/elves/Shaman.cfg	2012-01-31 11:41:12.469939672 +0100
--- editable/units/elves/Shaman.cfg	2012-01-31 11:41:13.292928324 +0100
***************
*** 20,26 ****
      usage=healer
      description= _ "Being partly faerie in nature, elves have an inherent capability for magic. This is realized in their affinity with the natural world, which they can call upon as an ally in combat. Enemies wandering in their forests may soon find themselves entangled by roots and hindered by the very ground they stand on.
  
! The healing abilities of the elves are also remarkable, and of capital use in battle."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_HEALS}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
--- 20,26 ----
      usage=healer
      description= _ "Being partly faerie in nature, elves have an inherent capability for magic. This is realized in their affinity with the natural world, which they can call upon as an ally in combat. Enemies wandering in their forests may soon find themselves entangled by roots and hindered by the very ground they stand on.
  
! The healing abilities of the elves are also remarkable, and of capital use in battle."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_HEALS}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
***************
*** 51,56 ****
--- 51,57 ----
          description=_"entangle"
          type=impact
          [specials]
+             {WEAPON_SPECIAL_MARKSMAN}
              {WEAPON_SPECIAL_SLOW}
          [/specials]
          damage=3
diff -rc cache/units/elves/Shyde.cfg editable/units/elves/Shyde.cfg
*** cache/units/elves/Shyde.cfg	2012-01-31 11:41:12.469939672 +0100
--- editable/units/elves/Shyde.cfg	2012-01-31 11:41:13.293928310 +0100
***************
*** 19,25 ****
      {AMLA_DEFAULT}
      cost=52
      usage=healer
!     description= _ "Devotion to the path of faerie may eventually transform an elf maiden into a creature of both worlds. Guided by a power which is little understood, these beautiful stewards of the elven forests epitomize the grace and mystery of their people."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_CURES}+{SPECIAL_NOTES_MAGICAL}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
--- 19,25 ----
      {AMLA_DEFAULT}
      cost=52
      usage=healer
!     description= _ "Devotion to the path of faerie may eventually transform an elf maiden into a creature of both worlds. Guided by a power which is little understood, these beautiful stewards of the elven forests epitomize the grace and mystery of their people."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_CURES}+{SPECIAL_NOTES_MAGICAL}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
***************
*** 54,59 ****
--- 54,60 ----
          description=_"ensnare"
          type=impact
          [specials]
+             {WEAPON_SPECIAL_MARKSMAN}
              {WEAPON_SPECIAL_SLOW}
          [/specials]
          damage=6
diff -rc cache/units/elves/Sorceress.cfg editable/units/elves/Sorceress.cfg
*** cache/units/elves/Sorceress.cfg	2012-01-31 11:41:12.470939659 +0100
--- editable/units/elves/Sorceress.cfg	2012-01-31 11:41:13.293928310 +0100
***************
*** 18,24 ****
      usage=mixed fighter
      description= _ "The world of faerie is far more potent than the corporeal world. Simply bringing some of this world into our own can have violent results. This is well-understood by the elves, although rarely used with malicious intent; doing so is no easy task, and is a very ill-regarded use of their strength.
  
! Those able to do so are roughly titled as ‘sorceresses’ by other races; and certainly are capable of acting the part."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_ARCANE}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
--- 18,24 ----
      usage=mixed fighter
      description= _ "The world of faerie is far more potent than the corporeal world. Simply bringing some of this world into our own can have violent results. This is well-understood by the elves, although rarely used with malicious intent; doing so is no easy task, and is a very ill-regarded use of their strength.
  
! Those able to do so are roughly titled as ‘sorceresses’ by other races; and certainly are capable of acting the part."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_MARKSMAN}+{SPECIAL_NOTES_SLOW}+{SPECIAL_NOTES_ARCANE}
      die_sound={SOUND_LIST:ELF_FEMALE_HIT}
      [portrait]
          size=400
***************
*** 50,55 ****
--- 50,56 ----
          description=_"entangle"
          type=impact
          [specials]
+             {WEAPON_SPECIAL_MARKSMAN}
              {WEAPON_SPECIAL_SLOW}
          [/specials]
          damage=4
diff -rc cache/units/humans/Loyalist_Lieutenant.cfg editable/units/humans/Loyalist_Lieutenant.cfg
*** cache/units/humans/Loyalist_Lieutenant.cfg	2012-01-31 11:41:12.471939645 +0100
--- editable/units/humans/Loyalist_Lieutenant.cfg	2012-01-31 11:41:13.293928310 +0100
***************
*** 8,16 ****
      profile="portraits/humans/transparent/lieutenant.png"
      {LEADING_ANIM "units/human-loyalists/lieutenant-lead-2.png" "units/human-loyalists/lieutenant-lead-1.png" -5,-28}
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=40
      movement_type=smallfoot
!     movement=6
      experience=80
      level=2
      alignment=lawful
--- 8,16 ----
      profile="portraits/humans/transparent/lieutenant.png"
      {LEADING_ANIM "units/human-loyalists/lieutenant-lead-2.png" "units/human-loyalists/lieutenant-lead-1.png" -5,-28}
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=48
      movement_type=smallfoot
!     movement=5
      experience=80
      level=2
      alignment=lawful
***************
*** 42,48 ****
          icon=attacks/sword-human.png
          type=blade
          range=melee
!         damage=8
          number=3
      [/attack]
      [attack]
--- 42,48 ----
          icon=attacks/sword-human.png
          type=blade
          range=melee
!         damage=9
          number=3
      [/attack]
      [attack]
***************
*** 51,58 ****
          icon=attacks/crossbow-human.png
          type=pierce
          range=ranged
!         damage=5
!         number=3
      [/attack]
      [idle_anim]
          {STANDARD_IDLE_FILTER}
--- 51,58 ----
          icon=attacks/crossbow-human.png
          type=pierce
          range=ranged
!         damage=6
!         number=2
      [/attack]
      [idle_anim]
          {STANDARD_IDLE_FILTER}
diff -rc cache/units/humans/Loyalist_Sergeant.cfg editable/units/humans/Loyalist_Sergeant.cfg
*** cache/units/humans/Loyalist_Sergeant.cfg	2012-01-31 11:41:12.472939631 +0100
--- editable/units/humans/Loyalist_Sergeant.cfg	2012-01-31 11:41:13.294928296 +0100
***************
*** 11,17 ****
      {MAGENTA_IS_THE_TEAM_COLOR}
      hitpoints=32
      movement_type=smallfoot
!     movement=6
      experience=32
      level=1
      alignment=lawful
--- 11,17 ----
      {MAGENTA_IS_THE_TEAM_COLOR}
      hitpoints=32
      movement_type=smallfoot
!     movement=5
      experience=32
      level=1
      alignment=lawful
diff -rc cache/units/humans/Mage_Red.cfg editable/units/humans/Mage_Red.cfg
*** cache/units/humans/Mage_Red.cfg	2012-01-31 11:41:12.471939645 +0100
--- editable/units/humans/Mage_Red.cfg	2012-01-31 11:41:13.294928296 +0100
***************
*** 12,18 ****
      movement=5
      experience=100
      level=2
!     alignment=neutral
      advances_to=Arch Mage,Silver Mage
      cost=40
      usage=mixed fighter
--- 12,18 ----
      movement=5
      experience=100
      level=2
!     alignment=lawful
      advances_to=Arch Mage,Silver Mage
      cost=40
      usage=mixed fighter
diff -rc cache/units/humans/Outlaw.cfg editable/units/humans/Outlaw.cfg
*** cache/units/humans/Outlaw.cfg	2012-01-31 11:41:12.473939617 +0100
--- editable/units/humans/Outlaw.cfg	2012-01-31 11:42:20.259006629 +0100
***************
*** 7,13 ****
      image="units/human-outlaws/outlaw.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      profile="portraits/humans/outlaw.png"
!     hitpoints=42
      movement_type=elusivefoot
      movement=7
      level=2
--- 7,13 ----
      image="units/human-outlaws/outlaw.png"
      {MAGENTA_IS_THE_TEAM_COLOR}
      profile="portraits/humans/outlaw.png"
!     hitpoints=49
      movement_type=elusivefoot
      movement=7
      level=2
***************
*** 16,24 ****
      advances_to=Fugitive
      cost=28
      usage=mixed fighter
!     description= _ "After some years of service, former ‘footpads’ rise up in the ranks of their fellow outlaws. Having proven themselves in combat, they are given more dangerous tasks, and a greater share of the spoils. Though many opponents would mock their choice of weaponry, the outlaws are well aware of its deadly capacity, and also of the ready availability of ammunition. Outlaws are somewhat ill at ease fighting during the day, preferring the cover of nightfall."
      die_sound={SOUND_LIST:HUMAN_DIE}
      {DEFENSE_ANIM "units/human-outlaws/outlaw-defend.png" "units/human-outlaws/outlaw.png" {SOUND_LIST:HUMAN_HIT} }
      [portrait]
          size=400
          side="left"
--- 16,27 ----
      advances_to=Fugitive
      cost=28
      usage=mixed fighter
!     description= _ "After some years of service, former ‘footpads’ rise up in the ranks of their fellow outlaws. Having proven themselves in combat, they are given more dangerous tasks, and a greater share of the spoils. Though many opponents would mock their choice of weaponry, the outlaws are well aware of its deadly capacity, and also of the ready availability of ammunition. Outlaws are somewhat ill at ease fighting during the day, preferring the cover of nightfall."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CONCEALMENT}
      die_sound={SOUND_LIST:HUMAN_DIE}
      {DEFENSE_ANIM "units/human-outlaws/outlaw-defend.png" "units/human-outlaws/outlaw.png" {SOUND_LIST:HUMAN_HIT} }
+     [abilities]
+         {ABILITY_CONCEALMENT}
+     [/abilities]
      [portrait]
          size=400
          side="left"
***************
*** 36,50 ****
          description=_"mace"
          type=impact
          range=melee
!         damage=8
!         number=2
      [/attack]
      [attack]
          name=sling
          description=_"sling"
          type=impact
          range=ranged
!         damage=6
          number=3
      [/attack]
      [attack_anim]
--- 39,53 ----
          description=_"mace"
          type=impact
          range=melee
!         damage=7
!         number=3
      [/attack]
      [attack]
          name=sling
          description=_"sling"
          type=impact
          range=ranged
!         damage=7
          number=3
      [/attack]
      [attack_anim]
diff -rc cache/units/humans/Outlaw_Footpad.cfg editable/units/humans/Outlaw_Footpad.cfg
*** cache/units/humans/Outlaw_Footpad.cfg	2012-01-31 11:41:12.472939631 +0100
--- editable/units/humans/Outlaw_Footpad.cfg	2012-01-31 11:41:13.295928282 +0100
***************
*** 39,45 ****
          icon=attacks/club-small.png
          type=impact
          range=melee
!         damage=4
          number=2
      [/attack]
      [attack]
--- 39,45 ----
          icon=attacks/club-small.png
          type=impact
          range=melee
!         damage=5
          number=2
      [/attack]
      [attack]
diff -rc cache/units/humans/Outlaw_Fugitive.cfg editable/units/humans/Outlaw_Fugitive.cfg
*** cache/units/humans/Outlaw_Fugitive.cfg	2012-01-31 11:41:12.473939617 +0100
--- editable/units/humans/Outlaw_Fugitive.cfg	2012-01-31 11:41:13.296928268 +0100
***************
*** 41,48 ****
          description= _ "mace"
          type=impact
          range=melee
!         damage=11
!         number=2
          icon=attacks/mace.png
      [/attack]
      [attack]
--- 41,48 ----
          description= _ "mace"
          type=impact
          range=melee
!         damage=7
!         number=3
          icon=attacks/mace.png
      [/attack]
      [attack]
***************
*** 51,57 ****
          type=impact
          range=ranged
          damage=7
!         number=4
          icon=attacks/sling.png
      [/attack]
      [attack_anim]
--- 51,57 ----
          type=impact
          range=ranged
          damage=7
!         number=3
          icon=attacks/sling.png
      [/attack]
      [attack_anim]
diff -rc cache/units/merfolk/Hunter.cfg editable/units/merfolk/Hunter.cfg
*** cache/units/merfolk/Hunter.cfg	2012-01-31 11:41:12.470939659 +0100
--- editable/units/merfolk/Hunter.cfg	2012-01-31 11:41:13.296928268 +0100
***************
*** 14,20 ****
      level=1
      alignment=lawful
      advances_to=Merman Spearman,Merman Netcaster
!     cost=15
      usage=mixed fighter
      description= _ "The skills employed by mermen in spear-fishing are easily translated into warfare, especially against those who are not at home in the water. In times of need, many mermen of that occupation will volunteer to swell the ranks of their military."
      die_sound=mermen-die.ogg
--- 14,20 ----
      level=1
      alignment=lawful
      advances_to=Merman Spearman,Merman Netcaster
!     cost=14
      usage=mixed fighter
      description= _ "The skills employed by mermen in spear-fishing are easily translated into warfare, especially against those who are not at home in the water. In times of need, many mermen of that occupation will volunteer to swell the ranks of their military."
      die_sound=mermen-die.ogg
diff -rc cache/units/orcs/Leader.cfg editable/units/orcs/Leader.cfg
*** cache/units/orcs/Leader.cfg	2012-01-31 11:41:12.471939645 +0100
--- editable/units/orcs/Leader.cfg	2012-01-31 11:41:13.297928254 +0100
***************
*** 8,16 ****
      {LEADING_ANIM "units/orcs/leader-lead-2.png" "units/orcs/leader-lead-1.png" 0,-30}
      ellipse="misc/ellipse"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=45
      movement_type=orcishfoot
!     movement=6
      experience=60
      level=1
      alignment=chaotic
--- 8,16 ----
      {LEADING_ANIM "units/orcs/leader-lead-2.png" "units/orcs/leader-lead-1.png" 0,-30}
      ellipse="misc/ellipse"
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=37
      movement_type=orcishfoot
!     movement=5
      experience=60
      level=1
      alignment=chaotic
***************
*** 51,58 ****
          icon=attacks/crossbow-orcish.png
          type=pierce
          range=ranged
!         damage=5
!         number=2
      [/attack]
      [attack_anim]
          [filter_attack]
--- 51,58 ----
          icon=attacks/crossbow-orcish.png
          type=pierce
          range=ranged
!         damage=3
!         number=1
      [/attack]
      [attack_anim]
          [filter_attack]
diff -rc cache/units/orcs/Ruler.cfg editable/units/orcs/Ruler.cfg
*** cache/units/orcs/Ruler.cfg	2012-01-31 11:41:12.471939645 +0100
--- editable/units/orcs/Ruler.cfg	2012-01-31 11:41:13.297928254 +0100
***************
*** 7,15 ****
      profile="portraits/orcs/leader.png"
      {LEADING_ANIM "units/orcs/ruler-lead-2.png" "units/orcs/ruler-lead-1.png" 0,-30}
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=60
      movement_type=orcishfoot
!     movement=6
      experience=120
      level=2
      alignment=chaotic
--- 7,15 ----
      profile="portraits/orcs/leader.png"
      {LEADING_ANIM "units/orcs/ruler-lead-2.png" "units/orcs/ruler-lead-1.png" 0,-30}
      {MAGENTA_IS_THE_TEAM_COLOR}
!     hitpoints=52
      movement_type=orcishfoot
!     movement=5
      experience=120
      level=2
      alignment=chaotic
***************
*** 41,47 ****
          icon=attacks/greatsword-orcish.png
          type=blade
          range=melee
!         damage=9
          number=3
      [/attack]
      [attack]
--- 41,47 ----
          icon=attacks/greatsword-orcish.png
          type=blade
          range=melee
!         damage=10
          number=3
      [/attack]
      [attack]
***************
*** 51,57 ****
          type=pierce
          range=ranged
          damage=6
!         number=3
      [/attack]
      [attack_anim]
          [filter_attack]
--- 51,57 ----
          type=pierce
          range=ranged
          damage=6
!         number=2
      [/attack]
      [attack_anim]
          [filter_attack]
diff -rc cache/units/trolls/Troll_Shaman.cfg editable/units/trolls/Troll_Shaman.cfg
*** cache/units/trolls/Troll_Shaman.cfg	2012-01-31 11:41:12.460939795 +0100
--- editable/units/trolls/Troll_Shaman.cfg	2012-01-31 11:41:13.298928240 +0100
***************
*** 44,50 ****
              {WEAPON_SPECIAL_MAGICAL}
          [/specials]
          range=ranged
!         damage=7
          number=3
          icon=attacks/fire-blast.png
      [/attack]
--- 44,50 ----
              {WEAPON_SPECIAL_MAGICAL}
          [/specials]
          range=ranged
!         damage=9
          number=3
          icon=attacks/fire-blast.png
      [/attack]
diff -rc cache/units/undead/Skele_Death_Knight.cfg editable/units/undead/Skele_Death_Knight.cfg
*** cache/units/undead/Skele_Death_Knight.cfg	2012-01-31 11:41:12.461939781 +0100
--- editable/units/undead/Skele_Death_Knight.cfg	2012-01-31 11:41:13.298928240 +0100
***************
*** 7,13 ****
      profile=portraits/undead/death-knight.png
      {MAGENTA_IS_THE_TEAM_COLOR}
      {LEADING_ANIM "units/undead-skeletal/deathknight-lead-2.png" "units/undead-skeletal/deathknight-lead-1.png" -17,-37}
!     hitpoints=66
      movement_type=undeadfoot
      movement=5
      experience=150
--- 7,13 ----
      profile=portraits/undead/death-knight.png
      {MAGENTA_IS_THE_TEAM_COLOR}
      {LEADING_ANIM "units/undead-skeletal/deathknight-lead-2.png" "units/undead-skeletal/deathknight-lead-1.png" -17,-37}
!     hitpoints=46
      movement_type=undeadfoot
      movement=5
      experience=150
***************
*** 26,33 ****
      description= _ "Tales are told of the mightiest warriors and generals, who, cursed with hate and stung by betrayal, have come back to this world as Death Knights. Wielding the same weapons as in life, they command the Undead in their quest for revenge."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}
      die_sound=lich-die.ogg
      [abilities]
!         {ABILITY_LEADERSHIP_LEVEL_3}
      [/abilities]
      [portrait]
          size=400
          side="left"
--- 26,38 ----
      description= _ "Tales are told of the mightiest warriors and generals, who, cursed with hate and stung by betrayal, have come back to this world as Death Knights. Wielding the same weapons as in life, they command the Undead in their quest for revenge."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}
      die_sound=lich-die.ogg
      [abilities]
!         {ABILITY_LEADERSHIP_LEVEL_2}
      [/abilities]
+     [resistance]
+         blade=60
+         impact=120
+         pierce=40
+     [/resistance]
      [portrait]
          size=400
          side="left"
***************
*** 46,53 ****
          icon=attacks/battleaxe-undead.png
          type=blade
          range=melee
!         damage=11
!         number=4
      [/attack]
      [attack]
          name=crossbow
--- 51,58 ----
          icon=attacks/battleaxe-undead.png
          type=blade
          range=melee
!         damage=8
!         number=3
      [/attack]
      [attack]
          name=crossbow
***************
*** 56,62 ****
          type=pierce
          range=ranged
          damage=6
!         number=2
      [/attack]
      {DEFENSE_ANIM_RANGE "units/undead-skeletal/deathknight-defend-2.png" "units/undead-skeletal/deathknight-defend-1.png" {SOUND_LIST:SKELETON_BIG_HIT} melee}
      {DEFENSE_ANIM_RANGE "units/undead-skeletal/deathknight-crossbow-defend.png" "units/undead-skeletal/deathknight-crossbow.png" {SOUND_LIST:SKELETON_BIG_HIT} ranged}
--- 61,67 ----
          type=pierce
          range=ranged
          damage=6
!         number=1
      [/attack]
      {DEFENSE_ANIM_RANGE "units/undead-skeletal/deathknight-defend-2.png" "units/undead-skeletal/deathknight-defend-1.png" {SOUND_LIST:SKELETON_BIG_HIT} melee}
      {DEFENSE_ANIM_RANGE "units/undead-skeletal/deathknight-crossbow-defend.png" "units/undead-skeletal/deathknight-crossbow.png" {SOUND_LIST:SKELETON_BIG_HIT} ranged}
diff -rc cache/units/undead/Skeleton_Archer.cfg editable/units/undead/Skeleton_Archer.cfg
*** cache/units/undead/Skeleton_Archer.cfg	2012-01-31 11:41:12.462939767 +0100
--- editable/units/undead/Skeleton_Archer.cfg	2012-01-31 11:41:13.299928226 +0100
***************
*** 38,44 ****
      [resistance]
          blade=60
          pierce=40
!         impact=120
      [/resistance]
      [attack]
          name=fist
--- 38,44 ----
      [resistance]
          blade=60
          pierce=40
!         impact=110
      [/resistance]
      [attack]
          name=fist
diff -rc cache/units/undead/Skeleton.cfg editable/units/undead/Skeleton.cfg
*** cache/units/undead/Skeleton.cfg	2012-01-31 11:41:12.461939781 +0100
--- editable/units/undead/Skeleton.cfg	2012-01-31 11:41:13.299928226 +0100
***************
*** 112,118 ****
      [resistance]
          blade=60
          pierce=40
!         impact=120
      [/resistance]
      [attack]
          name=axe
--- 112,118 ----
      [resistance]
          blade=60
          pierce=40
!         impact=110
      [/resistance]
      [attack]
          name=axe
diff -rc cache/units/undead/Spirit_Ghost.cfg editable/units/undead/Spirit_Ghost.cfg
*** cache/units/undead/Spirit_Ghost.cfg	2012-01-31 11:41:12.463939753 +0100
--- editable/units/undead/Spirit_Ghost.cfg	2012-01-31 11:41:13.299928226 +0100
***************
*** 14,20 ****
      level=1
      alignment=chaotic
      advances_to=Wraith,Shadow
!     cost=20
      usage=scout
      description= _ "Enslaved within a shroud of enchantments, a trapped spirit may be likened to the wind in the sails of a ship. This damned vessel becomes an unfailing servant which can be bound to whatever task their master sees fit."+{SPECIAL_NOTES}+{SPECIAL_NOTES_DRAIN}+{SPECIAL_NOTES_SPIRIT}+{SPECIAL_NOTES_ARCANE}
      die_sound=wail-long.wav
--- 14,20 ----
      level=1
      alignment=chaotic
      advances_to=Wraith,Shadow
!     cost=19
      usage=scout
      description= _ "Enslaved within a shroud of enchantments, a trapped spirit may be likened to the wind in the sails of a ship. This damned vessel becomes an unfailing servant which can be bound to whatever task their master sees fit."+{SPECIAL_NOTES}+{SPECIAL_NOTES_DRAIN}+{SPECIAL_NOTES_SPIRIT}+{SPECIAL_NOTES_ARCANE}
      die_sound=wail-long.wav
diff -rc cache/units/wose/Ancient.cfg editable/units/wose/Ancient.cfg
*** cache/units/wose/Ancient.cfg	2012-01-31 11:41:12.464939739 +0100
--- editable/units/wose/Ancient.cfg	2012-01-31 11:41:13.301928198 +0100
***************
*** 18,23 ****
--- 18,26 ----
      description= _ "There is a curious story of a farmer who had a great oak in his lawn, a tree which had stood when his father first broke the soil on that land, and under which his family had many memories. It was thus a shock to him, one morning, when he awoke to find it somehow spirited away, with nothing but a bare patch of earth where it once stood.
  
  Such encounters are all that is known of what are presumably the most ancient of woses."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}+{SPECIAL_NOTES_REGENERATES}
+     [resistance]
+         cold=100
+     [/resistance]
      [portrait]
          size=400
          side="left"
diff -rc cache/units/wose/Elder.cfg editable/units/wose/Elder.cfg
*** cache/units/wose/Elder.cfg	2012-01-31 11:41:12.463939753 +0100
--- editable/units/wose/Elder.cfg	2012-01-31 11:41:13.302928184 +0100
***************
*** 17,22 ****
--- 17,25 ----
      description= _ "Woses have been said to possess many shapes, all of them tree-like in form, and as they age, to increase in size. Tales tell of woses who resemble trees in this respect as well, towering over the creatures who walk beneath them. This is the greater part of why they are so rarely seen — standing in the slumber which they so often do, a wose of that stature appears to be nothing more than an oddly-shaped tree. Even a careless elf can sometimes be fooled.
  
  Woses are not warriors by any means, but their great strength can easily be turned to violence, should someone manage to raise the ire of these creatures."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}+{SPECIAL_NOTES_REGENERATES}
+     [resistance]
+         cold=100
+     [/resistance]
      [portrait]
          size=400
          side="left"
diff -rc cache/units/wose/Wose.cfg editable/units/wose/Wose.cfg
*** cache/units/wose/Wose.cfg	2012-01-31 11:41:12.463939753 +0100
--- editable/units/wose/Wose.cfg	2012-01-31 11:41:13.303928170 +0100
***************
*** 18,23 ****
--- 18,26 ----
      description= _ "Rarely seen even by elves, the Wose is an order of creature about which little is known. The elves are the source of most of this knowledge; they know that these beings are not descended from trees, despite the similarity in form, and they know that a wose is more closely tied to the faerie world than the elves themselves, though in a different way. The motives and workings of their kind are unknown, though most subscribe to the obvious theory that woses are dedicated wardens of the natural world.
  
  Woses are utterly unwarlike, but possess a great strength. They are, however, neither used to, nor quick at moving around."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}+{SPECIAL_NOTES_REGENERATES}
+     [resistance]
+         cold=100
+     [/resistance]
      [portrait]
          size=400
          side="left"
